src/scala/ly/stealth/mesos/kafka/Scheduler.scala

```diff
@@ -256,6 +296,28 @@ object Scheduler extends org.apache.mesos.Scheduler {
     }
 
     cluster.save()
+    pauseOrResumeOffers()
+  }
+
+  private def pauseOrResumeOffers(): Unit = {
+    val clusterIsSteadyState = cluster.getBrokers.asScala.forall(_.isSteadyState)
+    // If all brokers are steady state we can request mesos to stop sending offers.
+    if (!this.offersAreSuppressed && clusterIsSteadyState) {
+      if (canSuppressOffers && driver.suppressOffers() == Status.DRIVER_RUNNING) {
+        logger.info("Cluster is now stable, suppressing offers.")
+        this.offersAreSuppressed = true
+      }
```

Comment:
Should we log the case where canSuppressOffers == True but driver.suppressOffers() != Status.DRIVER_RUNNING?